src/controllers/followController.js

```diff
@@ -0,0 +1,206 @@
+import Models from '../db/models';
+import errorResponse from '../helpers/index';
+
+const { User, Follow } = Models;
+
+/**
+ * @description handles follow and unfollow
+ * @class FollowController
+ */
+class FollowController {
+  /**
+   * @description Handles the follow feature
+   * @param {object} request - Request object
+   * @param {object} response - Response object
+   * @returns {object} An object containing all the data related to the unfollowed user
```

Comment:
HI @frostyblok what's the difference between this line 15 for this method and line 57 for the next method?

Reply:
Right now, there are no differences. The implementation has a typographical error and I will look into fixing that as soon as possible. Thanks for noticing that!